frontend/app/openproject-app.js

```diff
@@ -203,7 +203,12 @@ openprojectApp
       $httpProvider.interceptors.push(function($q) {
         return {
           'request': function(config) {
-            if (!config.url.match('(^/templates|\\.html$)')) {
+            // OpenProject can run in a subpath e.g. https://mydomain/open_project.
+            // We append the path found as the base-tag value to all http requests
+            // to the server except:
+            //   * when the path is already appended
+            //   * when we are getting a template
+            if (!config.url.match('(^/templates|\\.html$|^' + window.appBasePath + ')')) {
```

Comment:
I currently wonder if this makes trouble for certain subfolder names. But those are probably esoteric cases (e.g. instance hosted under `https://mydomain/assets/`)